[FAU-409] Switch to @wordpress/scripts (#22)
RRZE-Webteam/FAU-Studium-Embed: resources/svg/README.md

# Usage

> This folder holds raw icons and generates a `sprite.svg` files out of them, If you are using an icon directly place it under `images` folder, as the files in this folder won't be available when they are built. (only `sprite.svg` file will be available after build).

Reviewer: Let's improve the wording according to the Boy Scout rule.
```suggestion
This folder holds raw icons and generates `sprite.svg` files from them. If you use an icon directly, place it in the `images/` folder, as the files in this folder won't be available when they are built. Only the `sprite.svg` file will be available after the build.
```
